### PR TITLE
Redone the changes made in PR #7894 which reversed by PR #7924

### PR DESCRIPTION
**Pull request by N-thony**
This PR request contains changes made in PR #7894 which was merged but reversed by PR #7924.
@lloyddewit we can merge this if everything is okay.

### CODE REVIEW

**Comment by lloyddewit** [comment]
@N-thony I can't see any changes

File: instat/Model/DataFrame/clsDataFramePage.vb [hunk 1]
```diff
@@ -272,9 +272,9 @@ Public Class clsDataFramePage
             columnHeader.strTypeShortCode = "(T)"
         ElseIf strHeaderType.Contains("logical") Then
             columnHeader.strTypeShortCode = "(L)"
-            ' Structured columns e.g. "circular" are coded with "(S)"
-        ElseIf strHeaderType.Contains("circular") Then
-            columnHeader.strTypeShortCode = "(S)"
+            ' Structured columns or Bigq or Bigz e.g. "circular" are coded with "(S)"
+        ElseIf strHeaderType.Contains("circular") OrElse strHeaderType.Contains("bigz") OrElse
+                       strHeaderType.Contains("bigq") Then
```
Comment (lloyddewit) [inline instat/Model/DataFrame/clsDataFramePage.vb]: should we set `columnHeader.strTypeShortCode`?
```suggestion
                       strHeaderType.Contains("bigq") Then
           columnHeader.strTypeShortCode = "(S)"
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@N-thony Thank you, looks good.

@berylwaswa If you can test/approve, then we can merge, thanks

**Comment by berylwaswa** [comment]
Okay, let me look at it.

Kind Regards,
Beryl Waswa.

**Review by berylwaswa — APPROVED** [review]
It's working fine.

**Pull request merged**